app/views/api/v2/hostgroups/main.json.rabl

```diff
@@ -3,6 +3,6 @@ object @hostgroup
 extends "api/v2/hostgroups/base"
 
 attributes :subnet_id, :subnet_name, :operatingsystem_id, :operatingsystem_name, :domain_id, :domain_name,
-           :environment_id, :environment_name, :compute_profile_id, :compute_profile_name, :ancestry,
+           :environment_id, :environment_name, :compute_profile_id, :compute_profile_name, :parent_id, :parent,
```

Comment:
I think `parent` should be `parent_name` for consistency, and since `#parent` is an AR object, adding a `#parent_name` method that returns a string could be used here instead, i.e. in NestedAncestryCommon.